### PR TITLE
common.xml: change NAV_SET_YAW_SPEED parameter attributes

### PR DESCRIPTION
**Pull request by arjun7965**
as per @rmackay9's [post](https://github.com/ArduPilot/ardupilot/issues/10803)

### CODE REVIEW

**Comment by arjun7965** [comment]
fixed @amilcarlucas :+1:

**Comment by arjun7965** [comment]
> Please also add increment="1" attribute.

Fixed

**Comment by amilcarlucas** [comment]
@arjun7965 Thanks, now please add a new PR, that adds the first two changes that I asked you to revert.

**Comment by arjun7965** [comment]
@amilcarlucas done. See #1120

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -1374,7 +1374,7 @@
         <description>Sets a desired vehicle turn angle and speed change.</description>
         <param index="1" label="Yaw" units="cdeg">Yaw angle to adjust steering by.</param>
         <param index="2" label="Speed" minValue="0" maxValue="1">speed - normalized to 0 .. 1</param>
-        <param index="3">Empty</param>
+        <param index="3" label="Angle" minValue="0" maxValue="1" increment="1">Final angle. (0=absolute, 1=relative)</param>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: @amilcarlucas @arjun7965 Just to be clear are you saying that:
- if index 3 = 0 the yaw param will be relative to the body frame (forward-facing-upright). 
- if index 3 = 1 the yaw will be relative to (added to) the original yaw.

IMO the label "Angle" and description "Final angle" do not convey this.

Perhaps:
- label: "Yaw reference"
- description: 1: Yaw param (1) is relative to the vehicle body frame. 2: Yaw param is relative to current/existing yaw when message received.

**Pull request merged**